chsdi/subscribers.py

```diff
@@ -61,8 +61,6 @@ def add_localizer(event):
     # TODO: clean-up when only Python 3.x and no longer 2.x is in use
     if not six.PY3:
         request.lang = request.lang.encode('ascii', 'ignore')
-    # The load balancer forwards requests as http, therefore we need to check X-Forwarded-Proto
-    request.scheme = request.headers.get('X-Forwarded-Proto', request.scheme)
```

Comment:
This was playing tricks on me and overriding the newly implemented request factory.
Deleting this should do the job @ltshb @boecklic

Reply:
That was a good example of bad code, changing the `request.scheme` inside a `add_localizer()` method. Once again it shows that good code with proper naming avoid headache and long debugging session.
@hansmannj very good job and finding, one step ahead to cleaner, better and more stable code :-)